### PR TITLE
Add property type map

### PR DESCRIPTION
**Pull request by krassowski**
See: https://github.com/davidhalter/jedi/commit/784e965d3a5449d0cd3cb7af233082cac5f3e530

### CODE REVIEW

**Comment by krassowski** [comment]
There will be also `namespace` type added in the future versions but it is not needed just yet.

File: jedi_language_server/type_map.py [hunk 1]
```diff
@@ -24,6 +25,7 @@
     "param": SymbolKind.Variable,
     "path": SymbolKind.File,
     "keyword": SymbolKind.Class,
+    "property": CompletionItemKind.Property,
```
Comment (pappasam) [inline jedi_language_server/type_map.py:28]:
```suggestion
    "property": SymbolKind.Property,
```

**Review by pappasam — APPROVED** [review]
lgtm!

**Comment by pappasam** [comment]
One unfortunate thing: based on my limited manual testing so far, it looks like this only works for completions. Jedi is still returning `function` for symbols.

**Pull request merged**